spec/support/features/fill_addresses_fields.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module FillAddressFields
+  def fill_addresses_fields_with(address)
+    fields = %w[
+      address1
+      city
+      zipcode
+      phone
+    ]
+    fields += if Spree::Config.use_combined_first_and_last_name_in_address
+      %w[name]
+    else
+      %w[firstmname lastname]
```

Comment:
Should this be `firstname` instead of `firstmname`? Also, the fact that the specs aren't failing because of the typo is not good I guess. 😛

Reply:
Thanks! I think specs will fail, I run them locally only against Solidus master, which uses the other branch of the conditional. Fixing this now.